### PR TITLE
ABR Whitelisting Functionality

### PR DESCRIPTION
**Pull request by nweber**
Whitelisting functionality and integration tests.

Testing issue: when running the integration tests, running all of them causes tests following the first to fail. However, running them all independently, they pass. I believe there is some lingering information on the DOM or the videoJS player itself. Calling dipose() and even removing the video object from the html does not seem to have any effect.

### CODE REVIEW

**Comment by dmlap** [comment]
Is it possible to move these additions out into a separate file?

File: src/js/videojs-dash.js [hunk 1]
```diff
@@ -1,6 +1,270 @@
+/*! videojs-contrib-dash - v1.1.2 - 2016-03-07
+ * Copyright (c) 2016 Brightcove  */
 (function(window, videojs, dashjs) {
   'use strict';
 
+  var WhitelistPlugin = function () {
+    var globalManifest;
```
Comment (dmlap) [inline src/js/videojs-dash.js]: We use one `var` declaration + commas [elsewhere in this project](https://github.com/nweber/videojs-contrib-dash/blob/1.x/src/js/videojs-dash.js#L304).

**Comment by mrocajr** [comment]
@nweber @dmlap 
Testing has been completed.  No issues have been found.

**Comment by dmlap** [comment]
@mrocajr do you know why the tests are still failing in Travis?

**Comment by mrocajr** [comment]
Looks like it's running against Linux...  I'll look into this further.

**Comment by dmlap** [comment]
The tests pass for me locally. @mrocajr I think we may have to turn off integration tests on Travis since it doesn't seem like all their machines have media source or h264 codec support (see my comments on #56)

**Comment by mrocajr** [comment]
I agree.  Maybe we should add them to the SVT regression.

**Comment by forbesjo** [comment]
The integration tests will still be run against master.

**Comment by forbesjo** [comment]
Closing in favor of #66

**Pull request closed without merging**